lib/main.js

```diff
@@ -10,14 +10,11 @@ import {DidKeyDriver} from './DidKeyDriver.js';
  * @param {object} options - Options hashmap.
  * @param {object} [options.verificationSuite=Ed25519VerificationKey2020] -
  *   Key suite for the signature verification key suite to use.
- * @param {Function} [options.convert] - An async function to convert a
- *   verification key pair (like ed25519) to a key-agreement key pair (such
- *   as x25519).
  *
  * @returns {DidKeyDriver} Returns an instance of a did:key resolver driver.
  */
-function driver({verificationSuite, convert} = {}) {
-  return new DidKeyDriver({verificationSuite, convert});
+function driver({verificationSuite} = {}) {
+  return new DidKeyDriver({verificationSuite});
```

Comment:
This looks like a breaking change to me, you're removing an optional parameter.  Existing code written to this API would no longer function.  Changelog indicates this is slated to be a minor release.

Reply:
@mattcollier right, that makes sense. This was just a cleanup though - the constructor doesn't actually have that parameter defined. (Meaning, this `driver()` function was passing it in, but nothing is receiving on the other end.) That's why I was thinking non breaking.

Reply:
OK, thanks for clarification.